ksqldb-serde/src/main/java/io/confluent/ksql/serde/avro/AvroFormat.java

```diff
@@ -83,4 +83,8 @@ protected ConnectSchemaTranslator getConnectSchemaTranslator(
     return new KsqlAvroSerdeFactory(schemaFullName)
         .createSerde(connectSchema, config, srFactory, targetType, isKey);
   }
+
+  public static String getKeySchemaName(final String name) {
+    return AvroProperties.AVRO_SCHEMA_NAMESPACE + "." + name + "_key";
```

Comment:
The linked issue (https://github.com/confluentinc/ksql/issues/6465) suggests camel casing -- any reason we're not doing that?

Reply:
Good call - at first I was worried that it would be hard to determine the original casing and convert it to Camel case  (e.g. `FOO_BAR` vs `FooBar` vs `FOOBAR` as the input) but if we're generating java classes we probably want to err on the side of aggressively turning it into java-friendly class names. I've added a test to demonstrate this